static/js/find-a-partner.js

```diff
@@ -190,5 +191,5 @@ YUI({
   populateTextbox();
 
 
-  Y.all('.search-not-run').removeClass('search-not-run');
+  Y.all('.p-find-a-partner search-not-run').removeClass('search-not-run');
```

Comment:
Js targeted classes should be prefixed with `js-` not p as they add functionally nor styling.